internal/packages/archetype/data_stream_test.go

```diff
@@ -80,5 +68,5 @@ func createAndCheckDataStream(t *testing.T, pd PackageDescriptor, dd DataStreamD
 	err = CreateDataStream(dd)
 	require.NoError(t, err)
 
-	checkPackage(t, pd.Manifest.Name, valid)
+	checkPackage(t, packageRoot, valid)
```

Comment:
Previously, it was working because of the `os.Chdir`, am I right?
In any case, better to use here `packageRoot` :+1: